diff -Naur gcc-3.3.2.orig/gcc/Makefile.in gcc-3.3.2/gcc/Makefile.in
--- gcc-3.3.2.orig/gcc/Makefile.in	2003-08-11 01:11:19.000000000 +0200
+++ gcc-3.3.2/gcc/Makefile.in	2004-03-07 04:50:56.000000000 +0100
@@ -2335,10 +2335,6 @@
 	rm -f include/limits.h
 	cp xlimits.h include/limits.h
 	chmod a+r include/limits.h
-# Install the README
-	rm -f include/README
-	cp $(srcdir)/README-fixinc include/README
-	chmod a+r include/README
 	$(STAMP) $@
 
 # fixinc.sh depends on this, not on specs directly.
@@ -2369,7 +2365,6 @@
 	(TARGET_MACHINE='$(target)'; srcdir=`cd $(srcdir); ${PWD_COMMAND}`; \
 	SHELL='$(SHELL)' ;\
 	export TARGET_MACHINE srcdir SHELL ; \
-	$(SHELL) ./fixinc.sh `${PWD_COMMAND}`/include $(SYSTEM_HEADER_DIR) $(OTHER_FIXINCLUDES_DIRS); \
 	rm -f include/syslimits.h; \
 	if [ -f include/limits.h ]; then \
 	  mv include/limits.h include/syslimits.h; \
diff -Naur gcc-3.3.2.orig/libiberty/Makefile.in gcc-3.3.2/libiberty/Makefile.in
--- gcc-3.3.2.orig/libiberty/Makefile.in	2003-03-23 11:16:03.000000000 +0100
+++ gcc-3.3.2/libiberty/Makefile.in	2004-03-07 04:51:06.000000000 +0100
@@ -260,7 +260,7 @@
 @MAINT@	echo stamp > stamp-functions
 
 INSTALL_DEST = @INSTALL_DEST@
-install: install_to_$(INSTALL_DEST) install-subdir
+install: install-subdir
 
 install_to_libdir: all
 	${mkinstalldirs} $(DESTDIR)$(libdir)$(MULTISUBDIR)
